src/libs/PaginationUtils.ts

```diff
@@ -124,7 +124,7 @@ function mergeAndSortContinuousPages<TResource>(sortedItems: TResource[], pages:
     const result = [sortedPages[0]];
     for (let i = 1; i < sortedPages.length; i++) {
         const page = sortedPages[i];
-        const prevPage = sortedPages[i - 1];
+        const prevPage = result[result.length - 1];
```

Comment:
prepage (`result[result.length - 1]`) could be same as page if the result contains only one page

Reply:
If result length is 1, then:
`page = sortedPages[0]`
and
`prevPage = sortedPages[0]`

page is same as prevPage which is misleading and could cause bugs

Reply:
Ah right! my bad